ci.Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM debian:bullseye-slim as builder
```

Comment:
Could this file be `Dockerfile-CI` so that it is next to the normal `Dockerfile` in the repo? The path in the `docker.yml` would need to be updated as well.